packages/xod-client/src/editor/components/inspectorWidgets/pinWidgets/StringPinWidget.jsx

```diff
@@ -38,10 +77,11 @@ const StringWidget = compose(
     },
   })
 )(props => {
+  const showQuotes = props.focused && props.isStringMode;
   const wrapperClassNames = cls('inspector-input-wrapper', {
-    'string-focused': props.focused,
+    'string-focused': showQuotes,
```

Comment:
```suggestion
    'with-fake-quotes': showQuotes,
```

Now `string-focused` becomes misleading a bit